dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/StatsTaskListGenerator.java

```diff
@@ -46,13 +48,15 @@ ImmutableList<Task<?>> createTasks(ConnectorArguments arguments) throws IOExcept
     return builder.build();
   }
 
-  enum Tool {
+  /** The source of performance statistics. */
+  enum Source {
```

Comment:
Optional: The `Source` word is very generic and thus easily confused with other concepts. I would suggest a two-word identifier to make it easier for future readers, e.g. `StatsSource`.

Reply:
Done, I renamed it to StatsSource